core/frontend/src/tile/map/MapFeatureInfo.ts

```diff
@@ -0,0 +1,35 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module MapLayers
+ */
+
+import { PropertyDescription, PropertyRecord, PropertyValue } from "@itwin/appui-abstract";
+import { Cartographic } from "@itwin/core-common";
+/** @alpha */
+export class MapFeatureInfoRecord extends PropertyRecord {
```

Comment:
Will there in future be a reason for this subclass to exist? Currently it adds nothing to `PropertyRecord`.

Reply:
I created it be future-proof, I expect new fields to be added, and I want to avoid the need to revisit every files where its being used.